include/wil/resource.h

```diff
@@ -5361,6 +5361,11 @@ namespace wil
     typedef unique_any<HTHEME, decltype(&::CloseThemeData), ::CloseThemeData> unique_htheme;
 #endif // __WIL_INC_UXTHEME
 
+#if defined(_INC_USERENV) && !defined(__WIL_INC_USERENV) && !defined(WIL_KERNEL_MODE)

```

Comment:
I think you need a check for `WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP | WINAPI_PARTITION_SYSTEM)` as well.

Reply:
Done, thanks.

Reply:
I look forward to the day when we can get rid of that entirely. :|